src/components/calcite-date-picker-month-header/calcite-date-picker-month-header.scss

```diff
@@ -1,137 +1,142 @@
 .header {
-  display: flex;
-  justify-content: space-between;
+  @apply flex justify-between;
+  //FIXME: Can this just be 4px? px-1
```

Comment:
yes, you need to also change the negative margin on the `chevron` to `-4px` as well